addons/procurement/schedulers.py

```diff
@@ -39,9 +43,23 @@ def run_scheduler(self, cr, uid, automatic=False, use_new_cursor=False, context=
         '''
         if use_new_cursor:
             use_new_cursor = cr.dbname
-        self._procure_confirm(cr, uid, use_new_cursor=use_new_cursor, context=context)
-        self._procure_orderpoint_confirm(cr, uid, automatic=automatic,\
-                use_new_cursor=use_new_cursor, context=context)
+        # Mail group to log execution
+        data_obj = self.pool['ir.model.data']
+        mail_group_obj = self.pool['mail.group']
+        __, mail_group_id = data_obj.get_object_reference(
+            cr, uid, 'procurement', 'group_mrp_scheduler_status')
```

Comment:
Get_object_reference will raise at this point if the code is updated and Odoo is restarted but the module is not updated with the new mail group record. Can you at least catch this error and work around it?

As is confirmed on the upstream PR, this should be unacceptible to Odoo 7.0. Can you follow up Olivier's advice to try again with a PR on master?